_posts/2019-04-25-subcontractors.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Working with subcontractors"
+date: 2019-04-25 09:00:00
+categories: ["bizdev"]
+author: "rdormer"
+---
+
+Out of all the problems an agency might face, "we have more opportunities than we can handle" is not something you'll typically hear anyone complaining about.  For those who are lucky enough to be in that position, it's always thought of as being "a good problem to have."  But make no mistake, having more opportunities than you can handle can be a real problem.  Aside from the obvious opportunity costs of all the work not taken, there are often good reasons why you may not be able to expand your permanent headcount just yet.
```

Comment:
`problem to have."` -> `problem to have".`

```suggestion
Out of all the problems an agency might face, "we have more opportunities than we can handle" is not something you'll typically hear anyone complaining about.  For those who are lucky enough to be in that position, it's always thought of as being "a good problem to have".  But make no mistake, having more opportunities than you can handle can be a real problem.  Aside from the obvious opportunity costs of all the work not taken, there are often good reasons why you may not be able to expand your permanent headcount just yet.
```

Reply:
The period is always placed inside of the quotation marks for American English :)